punjab/httpb.py

```diff
@@ -381,6 +381,7 @@ def render_POST(self, request):
             log.msg(str(request.content.read()))
             request.content.seek(0, 0)
 
+        self.service.forward_ip = request.getHeader("x-forwarded-for")
```

Comment:
There's also a new proposed header, not sure if we should consider it here as well? http://tools.ietf.org/html/rfc7239